schema/GeneticVariant_Alt_ID_Database_properties.mcf

```diff
@@ -0,0 +1,1401 @@
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database cHEO_Genetics_Diagnostic_Laboratory_Children's_Hospital_of_Eastern_Ontario for a genetic variant."
+
+Node: dcid:tuberous_sclerosis_database_(TSC2)_ID
+name: "tuberous_sclerosis_database_(TSC2)_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database tuberous_sclerosis_database_(TSC2) for a genetic variant."
+
+Node: dcid:claritas_Genomics_ID
+name: "claritas_Genomics_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database claritas_Genomics for a genetic variant."
+
+Node: dcid:deafness_Gene_Diagnosis_Xijing_Hospital_ID
+name: "deafness_Gene_Diagnosis_Xijing_Hospital_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database deafness_Gene_Diagnosis_Xijing_Hospital for a genetic variant."
+
+Node: dcid:leiden_Muscular_Dystrophy_(MYPN)_ID
+name: "leiden_Muscular_Dystrophy_(MYPN)_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database leiden_Muscular_Dystrophy_(MYPN) for a genetic variant."
+
+Node: dcid:dhiti_Omics_Technologies_Pvt._Ltd._ID
+name: "dhiti_Omics_Technologies_Pvt._Ltd._ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database dhiti_Omics_Technologies_Pvt._Ltd. for a genetic variant."
+
+Node: dcid:liping_Wei_Laboratory_Peking_University_ID
+name: "liping_Wei_Laboratory_Peking_University_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database liping_Wei_Laboratory_Peking_University for a genetic variant."
+
+Node: dcid:australian_Inherited_Retinal_Disease_Registry_&_DNA_Bank_Sir_Charles_Gairdner_Hospital_ID
+name: "australian_Inherited_Retinal_Disease_Registry_&_DNA_Bank_Sir_Charles_Gairdner_Hospital_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database australian_Inherited_Retinal_Disease_Registry_&_DNA_Bank_Sir_Charles_Gairdner_Hospital for a genetic variant."
+
+Node: dcid:neurogenetic_Laboratory_Oslo_University_Hospital_ID
+name: "neurogenetic_Laboratory_Oslo_University_Hospital_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database neurogenetic_Laboratory_Oslo_University_Hospital for a genetic variant."
+
+Node: dcid:athena_Diagnostics_Inc_ID
+name: "athena_Diagnostics_Inc_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database athena_Diagnostics_Inc for a genetic variant."
+
+Node: dcid:dST/NWU_Preclinical_Drug_Development_Platform_North-West_University_ID
```

Comment:
Remove '/' from property names?

Reply:
removed / and other problematic symbols from names